extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -31,7 +31,8 @@ require([
 		//Fallback to the generic playlist when no recommended videos playlist is set for the wiki
 		recommendedPlaylist = videoDetails.recommendedVideoPlaylist || 'Y2RWCKuS',
 		inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
-		willAutoplay = featuredVideoCookieService.isAutoplayEnabled() && inAutoplayCountries,
+		isAutoplayEnabled = featuredVideoCookieService.getAutoplay() !== '0',
+		willAutoplay = isAutoplayEnabled && inAutoplayCountries,
```

Comment:
If we don't use isAutoplayEnabled except for setting willAutoplay, I'd consider moving this whole logic to a method getWillAutoplay() so that it's nicely separated :)